test/index.test.js

```diff
@@ -1,8 +1,700 @@
+/* eslint-disable no-underscore-dangle */
 'use strict';
-const assert = require('chai').assert;
+const chai = require('chai');
+const assert = chai.assert;
+const mockery = require('mockery');
+const sinon = require('sinon');
+const joi = require('joi');
+
+chai.use(require('chai-as-promised'));
```

Comment:
I don't know if there is any point in including `chai-as-promised` as these tests do not utilize the added assertions.